src/Microsoft.Health.Dicom.Api/Features/BackgroundServices/DeletedInstanceCleanupWorker.cs

```diff
@@ -42,15 +44,17 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
         {
             try
             {
-                await Task.Delay(_pollingInterval, stoppingToken);
+                await Task.Delay(_options.PollingInterval, stoppingToken);
 
-                bool success;
-                int retrievedInstanceCount;
+                DeleteSummary summary;
                 do
                 {
-                    (success, retrievedInstanceCount) = await _deleteService.CleanupDeletedInstancesAsync(stoppingToken);
+                    summary = await _deleteService.CleanupDeletedInstancesAsync(stoppingToken);
+
+                    if (summary.Metrics != null)
```

Comment:
I would suggest writting metrics of delete time first, as they don't request as much logic, and if there is any error they still write